In [1]:
import numpy as np

Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

In [8]:
std = 16
a=0.05
M=80
n=256

In [9]:
t1=80-1.96*16/256**(1/2)
t1

78.04

In [10]:
t2=80+1.96*16/256**(1/2)
t2

81.96

Ответ: [78.04;81.96]

В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

In [2]:
x=np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [17]:
H=x.mean()
H

6.590000000000001

In [13]:
a=0.05

In [19]:
n=len(x)
n

10

In [25]:
sum =0;
for i in x:
    sum += (i-H)**2
sum=sum/(n-1)
sum

0.20322222222222233

In [31]:
std=sum**0.5

In [32]:
t1=H-(2.26216*std)/n**0.5
t1

6.2675154469473275

In [33]:
t2=H+(2.26216*std)/n**0.5
t2

6.912484553052674

Ответ: [6.27;6.91]

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

In [35]:
a=0.05
mu0=17
mu=17.5
n=100
D=4
s=D**0.5

In [37]:
Zp=(mu-mu0)/(s/n**0.5)
Zp

2.5

H0:mu=mu0
H1:mu>m0

In [39]:
Zt=1.645

Ответ:Принимаем гипотезу H1 с доверительной вероятностью 95%

Продавец утверждает, что средний вес пачки печенья составляет 200 г.
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [50]:
mu0=200
n=10
x=np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
a=0.01
mu=x.mean()

In [51]:
Tp=(mu-mu0)*n**0.5/((x.var(ddof=1)**0.5))
Tp

-1.0651074037450896

In [ ]:
Tt=3.25

Ответ:Принимаем гипотезу H0 с Уровнем значимости 1%